transactions/randomBeaconHistory/scripts/get_source_of_randomness_page.cdc

```diff
@@ -3,5 +3,5 @@ import "RandomBeaconHistory"
 /// Retrieves the source of randomness for the requested block height from the RandomBeaconHistory contract.
 ///
 access(all) fun main(page: UInt64, perPage: UInt64): RandomBeaconHistory.RandomSourceHistoryPage {
-    return RandomBeaconHistory.getRandomSourceHistoryPage(page: page, perPage: perPage)
+    return RandomBeaconHistory.getRandomSourceHistoryPage(page, perPage: perPage)
```

Comment:
cc @janezpodhostnik There was an error while testing this script, not how much important it is though.

Reply:
Its not that important, as it is mostly as an example, but we should fix this. I'll open up a PR.

Reply:
It is fixed in this PR, so not sure there's a need to open up another PR.